SDL_ttf.c

```diff
@@ -3350,7 +3364,7 @@ static SDL_Surface* TTF_Render_Wrapped_Internal(TTF_Font *font, const char *text
 
             if (numLines >= maxNumLines) {
                 char **saved = strLines;
-                maxNumLines += (width / wrapLength) + 1;
+                maxNumLines += (width / (wrapLength + 1)) + 1;
```

Comment:
Why are you making this change?

Reply:
if wrapLength is zero, then it would divide by 0 and crash, isn'it ?

Reply:
Yes, but it actually changes the maxNumLines calculation for certain values of width and wrapLength. Maybe instead you can explicitly check for wrapLength == 0?

Reply:
I don't think it matters, it is simply the increment for the memory in to realloc, so that we don't realloc for each new line pointers.

but, since we allow wrapLength == 0; maybe a we could just use a fixed increment. so that on don't increment of "width" in one shot.